Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -23,6 +23,11 @@
     state: pda
   - type: Pda
     state: pda
+    pAISlot:
+      priority: -2
+      whitelist:
+        tags:
+        - PAISlot
```

perhaps whitelisting entities with PAI component?
```suggestion
        components:
        - PAI
```